packages/flutter_tools/templates/list_detail_app/lib/src/app.dart.tmpl

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:{{projectName}}/src/dummy_feature/dummy_item_details_view.dart';
+import 'package:{{projectName}}/src/dummy_feature/dummy_item_list_view.dart';
+import 'package:{{projectName}}/src/settings/settings_controller.dart';
+import 'package:{{projectName}}/src/settings/settings_view.dart';
```

Comment:
These imports should probably be relative since they reference files form the same library?